xclim/sdba/adjustment.py

```diff
@@ -370,6 +382,13 @@ def adjust(
             sim = hist.copy()
             sim.attrs["_is_hist"] = True
 
+        # This below implies that ref.time and sim.time have the same size
+        # Since `ref,hist, sim` are in the same `map_groups` call, they must have
+        # the same time
+        if cls._replace_sim_time:
+            sim_time = sim.time
+            sim["time"] = ref["time"]
```

Comment:
This would only work with a uniform calendar, no ? Should we raise a meaningful error if the sizes and/or calendars are different ?